### PR TITLE
Fixed parsing db name in slowlog

### PR DESCRIPTION
**Pull request by askomorokhov**
@percona-csalguero, please review this PR.

### CODE REVIEW

**Comment by percona-csalguero** [comment]
In master branch:
```
go test
OK: 22 passed
PASS
ok      github.com/percona/go-mysql/log/slow    0.008s
```
in this branch
```
go test
OOPS: 18 passed, 4 FAILED
--- FAIL: Test (0.01s)
FAIL
exit status 1
```

File: log/slow/parser.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ import (
 // Regular expressions to match important lines in slow log.
 var timeRe = regexp.MustCompile(`Time: (\S+\s{1,2}\S+)`)
 var userRe = regexp.MustCompile(`User@Host: ([^\[]+|\[[^[]+\]).*?@ (\S*) \[(.*)\]`)
+var schema = regexp.MustCompile(`Schema: (.+) Last_errno:`)
```
Comment (percona-csalguero) [inline log/slow/parser.go]: The regexp should be:
```
var schema = regexp.MustCompile(`Schema: *(\S+) *Last_errno:`)
```
to not to have spaces in the schema name. This also fixes all the tests:
```
 go test
OK: 22 passed
PASS
ok      github.com/percona/go-mysql/log/slow    0.008s
```

Comment (askomorokhov) [inline log/slow/parser.go]: Fixed, with considering that db name can contain space
```
+------------------------------------------------------------------+
| Database                                                         |
+------------------------------------------------------------------+
| information_schema                                               |
| aa aa                                                            |
| mysql                                                            |
```

**Comment by askomorokhov** [comment]
tests passed
```
go test
OK: 22 passed
PASS
ok      github.com/percona/go-mysql/log/slow    0.013s

```

**Comment by percona-csalguero** [comment]
:+1: but we should add specific tests

**Pull request merged**